src/pipeline.ts

```diff
@@ -185,16 +187,28 @@ export async function checkNewCandidateMatch(
 	return result === InjectionResult.SUCCESS || result === SaveResult.SAVED;
 }
 
-async function findSearchableTorrents() {
-	const { torrents } = getRuntimeConfig();
+async function findSearchableTorrents(useData : boolean) {
+	const { torrents, dataDirs } = getRuntimeConfig();
 	let parsedTorrents: Searchee[];
-	if (Array.isArray(torrents)) {
-		const searcheeResults = await Promise.all(
-			torrents.map(createSearcheeFromTorrentFile)
-		);
+	if (useData) {
+		const fullPaths = [];
```

Comment:
can we extract this new code to a helper in a different file (maybe searchee.ts), a la loadTorrentDirLight? I like to keep new code to a minimum in pipeline.js because I think it's already pretty messy